CMakeLists.txt

```diff
@@ -50,6 +50,19 @@ if(CHECK_PIE_SUPPORTED)
   endif()
 endif()
 
+if(PROVISION_DEPENDENCIES)
+  # Run script to provision dependencies.
+  find_program (BASH_PROGRAM bash)
+  if(BASH_PROGRAM)
+    execute_process(
```

Comment:
It is probably fine not to print anything indicating that we start downloading dependencies here, because the deps.sh script is verbobse enough and prints something before downloading, so that the user doesn't have to wait in an uninformed state.